CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change Log
 
+## 2021.07.23
+- Initial release, original SnowEx hackweek content 🎉
```

Comment:
can remove, perhaps add comment with link to https://keepachangelog.com

Reply:
Since you've been with this template from the start, could you populate the changelog for this PR so it's more useful than "added updates from SnowEx to template"? I've taken out these lines and given it the [keepachangelog](https://keepachangelog.com/) infrastructure.

Reply:
i can modify after we merge this, then add a tag